<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>warpingForm</class>
 <widget class="QWidget" name="warpingForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>735</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>681</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>A</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="inputwarpingForma">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimum">
       <double>-1.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>B</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="inputwarpingFormb">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimum">
       <double>-1.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>C</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="inputwarpingFormc">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimum">
       <double>-1.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>D</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="inputwarpingFormd">
      <property name="minimum">
       <double>-1.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>681</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>E</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="inputwarpingForme">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimum">
       <double>-1.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>F</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="inputwarpingFormf">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimum">
       <double>-1.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>G</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="inputwarpingFormg">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimum">
       <double>-1.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>H</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="inputwarpingFormh">
      <property name="minimum">
       <double>-1.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>691</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Esta função altera a geometria e, eventualmente, a posição da imagem. Altere os valores de a,b,c,d,e,f,g,h para controlar o template.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
